spec.bs

```diff
@@ -4762,6 +4787,18 @@ To <dfn>update storage metrics</dfn> given [=per participant metrics=] |metrics|
 
 </div>
 
+<div algorithm>
+To <dfn>update cumulative buyer time</dfn> given [=per participant metrics=] |metrics| and a

```

Comment:
nit: maybe "update cumulative buyer time metrics" to make it clearer that it's for metrics.

Reply:
Done